src/components/atom/AgeGroupItem.tsx

```diff
@@ -0,0 +1,17 @@
+import {FC} from "react";
```

Comment:
Why are you using FC here? We don't really use it in the project. To keep it consistent, I would avoid using it. Do you think it provides value in this situation?